src/content/docs/journey-large-logs/organize.mdx

```diff
@@ -0,0 +1,196 @@
+6. Click Create to save your new partition.
+
+    </Side>
+    <Side>
+
+<img
+    title="log-partition"
+    alt="An image displaying New Relic's log partion UI"
+    src={logsPartition}
+/>
+
+    </Side>
+</SideBySide>
+
+
+
+
+This creates a data partition with standard data retention for all Java logs. They would follow the same steps for their infrastrucutre logs, only changing the retention to secondary and query to `logtype=infrastructure`.
+
+    </Step>
+    <Step>
+
+### Parse your logs
+
+To parse your logs:
+
+<SideBySide>
+    <Side>
+
+1. From **Manage Data** on the left nav of the logs UI, click **Parsing**, then click **Create parsing rule**.
+2. Enter a name for the new parsing rule.
+3. Select an existing field to parse (default = message), or enter a new field name.
+4. Enter a valid NRQL WHERE clause to match the logs you want to parse. 
+5. Select a matching log if one exists, or click on the paste log tab to paste in a sample log.
+6. Enter the parsing rule and validate it's working by viewing the results in the Output section. (See example below)
+7. Enable and save the custom parsing rule.
+
+    </Side>
+    <Side>
+
+<img
+    title="log-parsing"
+    alt="An image displaying New Relic's log parsing UI"
+    src={logsParsing}
+/>
+
+    </Side>
+</SideBySide>
+
+
+
+The following example guides you through a specific example on creating a parsing rule:
+
+<CollapserGroup>
+    <Collapser
+        id="example"
+        title="Parsing logs example"
+    >
+Let's work with the example we used earlier in this doc. You have logs that follow this pattern:
+
+```
+{
+    "message": "32 4329 store_Portland"
+}
+```
+
+You want to pull out the transaction amount, order number, and store location. Parsing rules are built using Grok, which uses the following pattern: `%{SYNTAX:SEMANTIC}`. `SYNTAX` is the pattern used to find the text and `SEMANTIC` is the identifier or attribute given to the matched result.
+
+In this case, our parsing rule would look like:
+
+```
+%{INT:transaction_total} %{INT:purchase_number} store%{DATA:store_location}
+```
+
+Once the parsing rule is created with the pattern above, it will return logs in the following manner:
+
+```
+{
+    "transaction_total": "32",
+    "purchase_number": "4329",
+    "store_location": "store_Portland",
+}
+```
+    </Collapser>
+</CollapserGroup>
+
+For a more in-depth look at creating Grok patterns to parse logs, [read our blog post](https://newrelic.com/blog/how-to-relic/how-to-use-grok-log-parsing).
+    </Step>
+</Steps>
+
+## What's next
+
+Congratulations on setting your organization up for success in managing logs! As your system grows and you ingest, you'll need to ensure an upkeep of parsing rules and partitions. If you're interested in diving deeper on what New Relic log management can do for you, check out these docs:
+
+* [Parsing log data](/docs/logs/ui-data/parsing): a deeper look into parsing logs.
```

Comment:
the rest of these bullets have lots of great context on what awaits in the linked doc, can we do that here?